spaces/views.py

```diff
@@ -150,6 +150,37 @@ def space_members(request, pk):
 
 @login_required
 def my_spaces_list(request):
+    interest_list = [
+        "Automotive",
+        "Beauty",
+        "Book and Literature",
+        "Business",
+        "Education",
+        "Events",
+        "Gaming",
+        "Health",
+        "Food and drink",
+        "Politics",
+        "Movies",
+        "Music",
+    ]
+
+    # Create or get existing interests
+    for interest in interest_list:
+        Interest.objects.get_or_create(name=interest)
```

Comment:
They don't create new interests since this loops over only the existing ones.

Reply:
Interests are only displayed in the spaces list. I could not get the logic how these interests are related to the space creation part?

Reply:
In the first draft of recommendation, popularity index ( counting numbers of posts etc. ) were given in the space list. And the mock up ( please view it in wiki ) shows recomendation and interest part will be given in my spaces page. After user selects the interest ( tags will be implemented in issue #125 ) recommendation will be updated.